src/test/resources/org/kohsuke/github/GHDiscussionTest/wiremock/testCreatedDiscussion/mappings/orgs_hub4j-test-org_teams_dummy-team_discussions-3.json

```diff
@@ -0,0 +1,47 @@
+{
+  "id": "00d7317e-48cb-4f70-9efb-148d34750fdb",
+  "name": "orgs_hub4j-test-org_teams_dummy-team_discussions",
+  "request": {
+    "url": "/orgs/hub4j-test-org/teams/dummy-team/discussions",
+    "method": "POST",
+    "headers": {
+      "Accept": {
+        "equalTo": "text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2"
+      }
+    },
+    "bodyPatterns": [
+      {
+        "equalToJson": "{\"title\":\"Dummy\",\"body\":\"This is a dummy discussion\"}",
+        "ignoreArrayOrder": true,
+        "ignoreExtraElements": true
+      }
+    ]
+  },
+  "response": {
+    "status": 404,
```

Comment:
@cmoulliard 
You're getting a "Not found" because you recorded a "Not Found".  😄 

Delete the stubs and take a new snapshot - it is easier than manually adjusting these files.

Reply:
I manually fixed the json file as taking a new snapshot was not working